tests/evergreen/test_task.py

```diff
@@ -8,6 +8,19 @@
 
 
 class TestTask(object):
+    def test_evg_task_has_estimated_cost(self, sample_task):
+        assert sample_task["estimated_cost"]
+        task = Task(sample_task, None)
+        assert task.estimated_cost
+
+    def test_evg_task_has_no_estimated_cost(self, sample_task):
+        del sample_task["estimated_cost"]
+        with pytest.raises(KeyError):
+            assert sample_task["estimated_cost"]
+        task = Task(sample_task, None)
+        with pytest.raises(AttributeError):
+            assert task.estimated_cost
+
```

Comment:
Not sure about these tests. Due to the inheritance of `Task` from `_BaseEvergreenObject`, if an `estimated_cost` field exists on the input json object, it will still be stored on the base object and retrievable via the base object `__getattr__` method (line 14). These tests just ensure that a `Task` object can be created properly -- with or without the `estimated_cost` field.

Reply:
Can't we delete the "estimated_cost" from the sample json?

Reply:
To mimic the current Task as closely as possible, I didn't delete it from the sample task json bc Evergreen is not immediately removing the field. It is going to return 0 for now & will eventually be removed in the future. I thought that this would test both scenarios.

Am I good to just delete the "estimated_cost" field?

Reply:
The evergreen change to remove the field is out for review right now. So I think we are safe removing it from the sample data. It will also give us a chance to make sure no other parts of the code are unexpectedly broken by the change.